src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.Key.cs

```diff
@@ -19,37 +16,112 @@ public sealed partial class ECDsaCng : ECDsa
         ///     object will not be the same object as the CngKey passed to the ECDsaCng constructor if that
         ///     constructor was used, however it will point at the same CNG key.
         /// </summary>
+        /// <remarks>                
```

Comment:
I removed it. There are better places for it like in the create and ctor methods that take keysize. However, I didn't attempt to re-add the comment to those places